resotoshell/resotoshell/__main__.py

```diff
@@ -194,7 +194,7 @@ def decode(value: Union[str, bytes]) -> str:
             handle_result(part, num == 0)
 
 
-@cache
+@lru_cache()
```

Comment:
This is not 1:1 replacement. To follow the `@cache` behaviour it needs to be `lru_cache(maxsize=None)`

Reply:
Since we only cache exactly one element - it should not matter?

Reply:
not really, except it is faster due to not having the cache size checks